tests/test_logging.py

```diff
@@ -77,35 +79,35 @@ def test_number_of_handlers(self):
         self.assertEqual(1, len(logger.handlers))
 
     def test_debug(self):
-        message = 'test debug message'
-        self.assertLogFormat(message, 'DEBUG')
+        message = "test debug message"
+        self.assertLogFormat(message, "DEBUG")
         self.assertLogFormat(message, logging.DEBUG)
 
     def test_info(self):
-        message = 'test info message'
-        self.assertLogFormat(message, 'INFO')
+        message = "test info message"
+        self.assertLogFormat(message, "INFO")
         self.assertLogFormat(message, logging.INFO)
 
     def test_warning(self):
-        message = 'test warning message'
-        self.assertLogFormat(message, 'WARNING')
+        message = "test warning message"
+        self.assertLogFormat(message, "WARNING")
         self.assertLogFormat(message, logging.WARNING)
 
     def test_error(self):
-        message = 'test error message'
-        self.assertLogFormat(message, 'ERROR')
+        message = "test error message"
+        self.assertLogFormat(message, "ERROR")
         self.assertLogFormat(message, logging.ERROR)
 
     def test_critical(self):
-        message = 'test critical message'
-        self.assertLogFormat(message, 'CRITICAL')
+        message = "test critical message"
+        self.assertLogFormat(message, "CRITICAL")
         self.assertLogFormat(message, logging.CRITICAL)
 
     def test_color(self):
-        message = 'test color message'
+        message = "test color message"
         with self.assertLogs(logger=self.logger_name) as cm:
             self.logger.warning(message)
             self.handler.flush()
 
             log_message = self.handler.format(cm.records[-1]).strip()
```

Comment:
On the last PR, I think you had something different here.
`record=cm.records[-1]`
Is there a reason it's different this time?

Reply:
No particular reason, it was just a leftover from my debugging process.